src/components/molecules/webcam-capture/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react'
+import Webcam from 'react-webcam'
+import PropTypes from 'prop-types'
+
+import Button from '_atoms/button'
+
+import styles from './styles.css'
+
+class WebcamCapture extends Component {
+  setRef = webcam => {
+    this.webcam = webcam
+  }
+
+  capture = () => {
+    const imageSrc = this.webcam.getScreenshot()
+    this.props.onImageCapturedFromWebcam(imageSrc)
+  }
+
+  render() {
+    const { height, width, scrennshotFormat, videoConstraints } = this.props.userMedia
+    return (
+      <div className={styles.webcamContainer}>
+        <Webcam
+          audio={false}
+          height={height}
+          width={width}
+          ref={this.setRef}
+          screenshotFormat={scrennshotFormat}
+          videoConstraints={videoConstraints}
+        />
+        <Button onClick={this.capture}>Capture</Button>
+      </div>
+    )
+  }
+}
+
+WebcamCapture.propTypes = {
+  onImageCapturedFromWebcam: PropTypes.func.isRequired,
```

Comment:
Overly descriptive props. We're already using the `WebcamCapture` component. This prop could be something like `onImageCaptured` only.